src/test/scala/com/campudus/tableaux/testtools/RequestCreation.scala

```diff
@@ -140,4 +140,20 @@ object RequestCreation {
     override def getJson: JsonObject = column.getJson.mergeIn(Json.obj("frontendReadOnly" -> true))
   }
 
+  case class FormattedGroupCol(name: String, groups: Seq[ColumnId], formatPattern: String) extends ColumnType("group") {
```

Comment:
Ich würde hier lieber ein `sealed abstract class BaseGroupCol` einführen, welches sich dann um `groups` in einer `getJson` kümmert. Dann könnte `GroupCol` und `FormattedGroupCol` davon Gebrauch machen.